packages/@guardian/source-react-components-development-kitchen/src/components/toggle-switch/styles.ts

```diff
@@ -86,6 +87,50 @@ export const slimStyles = css`
 	}
 `;
 
+export const webStyles = css`
+	width: 2.75rem;
+	height: 1.5rem;
+	border-radius: 15.5px;
+
+	&:before {
+		content: '';
+		position: absolute;
+		top: 6px;
+		height: 8px;
+		width: 4px;
```

Comment:
Should these be rem?

Reply:
Looking about the rest of Source, most things are in `px`. @SiAdcock do you have a preference?

Reply:
if the component's height is set in rem these should probably be too? other wise the component will respond to global text size but the pseudo potentially won't

Reply:
Yeah updated, thanks for the explanation!